### ISSUE TITLE
Extension not working (chrome)

### ISSUE THREAD

**Issue by thecko** [Q42/TrelloScrum#122]
Maybe trello made changes but the extension ceased working, no errors on dev tools, so I assume it's trello's changes.

**Comment by Swandog** [comment]
Has stopped working for me on both Firefox and Chrome

**Comment by Swandog** [comment]
If I had to guess I'd say it's that Trello changed their HTML/CSS...I went into the debugger and it's doing `$('.list')`, and there aren't any elements on the page with that class.

My guess is that the new selector should be `$( "[data-testid='list']" )`, but TBH I don't know this extension or Javascript well enough to say for sure.

**Comment by samthomson** [comment]
same here. Any solution to this?

**Comment by jesslancaster** [comment]
Temporary work around: 
Go to https://trello.com/labs and disable the option “Board experience enhancements”

This will revert back to the old Trello and allow extensions to work as they did before.

**Comment by considermycat** [comment]
Hi, the workaround has stopped working for me on Chrome 119: Trello no longer offers that toggle on the Labs page, presumably because the relevant code has now reached full release. Weirdly, the extension still works on Edge.

**Comment by venturivenn** [comment]
> Hi, the workaround has stopped working for me on Chrome 119: Trello no longer offers that toggle on the Labs page, presumably because the relevant code has now reached full release. Weirdly, the extension still works on Edge.

HI, How did you install on chrome? For me, using the chrome extension on edge, didn´t work.

**Comment by considermycat** [comment]
@venturivenn Hi, yeah, it's stopped working for me on Edge now, too. I guess the old Trello code was still cached. The extension is installed from the Chrome Web Store on both browsers (Edge and Chrome)

**Comment by venturivenn** [comment]
I think Atalassian is forncing us to use some of the powers up. But, until the moment, I didint see any similar option. All ouo automation and process is based os points on title, not in custom fillds. 

> @venturivenn Hi, yeah, it's stopped working for me on Edge now, too. I guess the old Trello code was still cached. The extension is installed from the Chrome Web Store on both browsers (Edge and Chrome)

I think Atlassian is forcing us to use some of their Power-Ups. But, so far, I haven't seen any similar option. All our automation and processes are based on points in the title, not in custom fields.

**Comment by samthomson** [comment]
So far I've started using the 'Sum Up' power up which is the closest (but not at all perfect) replacement out of those I've tried. I also tried 'Agile tools by Corrello' (doesn't seem to do anything) and 'Story points for trello' (so slow as to be unusable).
If anyone finds a better replacement to this great extension, please share :)

**Comment by Swandog** [comment]
OK, I spent a few hours today debugging and fixing the extension, I think. I put my changes in a fork of this repo:

https://github.com/Swandog/TrelloScrum

You can follow the instructions there to install my changes as an unpacked extension. I think they will work, but I literally only tried it for me. You'll probably have to disable and/or remove this version, too.

I'm putting up a pull request to this repo, so hopefully the devs can take my changes and use them to update the Webstore version. I'm not planning on publishing this in the webstore myself.

I don't have a fix for the Firefox version, as I haven't found a repo for that, so I haven't tried (yet).

**Comment by venturivenn** [comment]
> OK, I spent a few hours today debugging and fixing the extension, I think. I put my changes in a fork of this repo:
> 
> https://github.com/Swandog/TrelloScrum
> 
> You can follow the instructions there to install my changes as an unpacked extension. I think they will work, but I literally only tried it for me. You'll probably have to disable and/or remove this version, too.
> 
> I'm putting up a pull request to this repo, so hopefully the devs can take my changes and use them to update the Webstore version. I'm not planning on publishing this in the webstore myself.
> 
> I don't have a fix for the Firefox version, as I haven't found a repo for that, so I haven't tried (yet).

It works for me. Thank you very much. You saved a lot of hours here.

**Comment by considermycat** [comment]
> OK, I spent a few hours today debugging and fixing the extension, I think. I put my changes in a fork of this repo:
> 
> https://github.com/Swandog/TrelloScrum
> 
> You can follow the instructions there to install my changes as an unpacked extension. I think they will work, but I literally only tried it for me. You'll probably have to disable and/or remove this version, too.
> 
> I'm putting up a pull request to this repo, so hopefully the devs can take my changes and use them to update the Webstore version. I'm not planning on publishing this in the webstore myself.
> 
> I don't have a fix for the Firefox version, as I haven't found a repo for that, so I haven't tried (yet).

Wow, thank you for doing this! Looks like our company IT blocks installation of extensions in this way, so hopefully your pull request will make its way into the webstore version before too long :-)

**Comment by samthomson** [comment]
@Swandog +1 works for me, thanks so much.

**Comment by Swandog** [comment]
@considermycat I realized that I missed a step in my installation instructions--you have to turn on "Developer Mode". I've added that to my README, but maybe that's why you weren't able to load it?

**Comment by considermycat** [comment]
@Swandog That's worked! Thank you!

**Comment by garethjmsaunders** [comment]
Thanks @Swandog. I've just forked your fork and made a few changes

[https://github.com/garethjmsaunders/scrum-for-trello](https://github.com/garethjmsaunders/scrum-for-trello)

I made the following changes:

1. SETTINGS BUTTON: I couldn't see the Settings button, so I have restored that, and made the button more visible by creating a white/greyscale version. I also added a text label "Scrum" which disappears in line with the other buttons' labels.
2. BOARD TOTAL: I've restored the board total too, and moved it to the far left of the buttons.
3. LIST TOTAL: The column totals for estimated points and consumed points are now displayed in the same order as they are on the cards. This has bugged me for as long as I have used the plugin.
4. JQUERY: I have upgraded the jQuery library from v2.1.4 to the latest in the v.2.x.x branch, v2.2.4.
5. PLANNING POKER POINTS: I also changed the default story points values from Fibonacci sequence (1, 2, 3, 5, 8, 13) to the Planning Poker™ values (0, 0.5, 1, 2, 3, 5, 8, 13, 20, 40, 100) as that is what I use!
6. SETTINGS POP-UP: I made a few subtle tweaks to the Settings pop-up to make it a little more consistent with other plugins’ settings panels while keeping the charm of the original.
